includes/admin/payments/refunds.php

```diff
@@ -166,20 +166,20 @@ function edd_refund_details_notes( $refund ) {
 }
 
 /**
- * Outputs Refund Attribuets box.
+ * Outputs "Refund Details" box.
  *
  * @since 3.0
  *
  * @param \EDD\Orders\Order $refund Current Refund.
  */
-function edd_refund_details_attributes( $refund ) {
+function edd_refund_details_details( $refund ) {
```

Comment:
There are currently not many details here, which is fine if none are needed.

However, one I would request is a line item referencing the original order **number** (and possibly date). I know the button on the screen takes me to the original order, but in my install, the refund order numbers are quite different from the original order numbers.

It would also be nice if the details could show whether this represents a full or partial refund of the original order.

Reply:
> It would also be nice if the details could show whether this represents a full or partial refund of the original order.

Maybe this Refund screen would be good place to have the "Related Refunds" box that was originally shown on the original Order.